assets/src/edit-story/components/canvas/useFocusCanvas.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback } from 'react';
+
+function useFocusCanvas() {
+  const focusCanvas = useCallback(() => {
+    setTimeout(() => {
+      const evt = new window.FocusEvent('focusout');
+      window.document.dispatchEvent(evt);
+    });
+  }, []);
+  return focusCanvas();
```

Comment:
Did you mean to call this function here? I'd just expect `return focusCanvas;`...

Reply:
🤦‍♂